lib/slime/compiler.ex

```diff
@@ -33,42 +41,58 @@ defmodule Slime.Compiler do
     ~s( #{to_string(name)}="#{value}")
   end
 
-  defp render_branch(%{type: :doctype, content: text}), do: text
-  defp render_branch(%{type: :text, content: text}), do: text
-  defp render_branch(%{} = branch) do
-    opening = branch.attributes
-              |> Enum.map(fn {k, v} -> render_attribute(k, v) end)
-              |> Enum.join
-              |> open(branch)
 
+  defp render_branch(%{type: :doctype, content: text}) do
```

Comment:
Is it dogma forcing `do-end` style over `do:` style? I don`t see why we should not use `do:` style.